Source/MOLXPCConnection/MOLXPCConnection.h

```diff
@@ -113,9 +120,14 @@
 @property(readonly, nonatomic, nullable) id remoteObjectProxy;
 
 /**
- The interface this object exports. (server)
+ The privileged interface this object exports. (server)
+ */
+@property(retain, nullable) NSXPCInterface *privilegedExportedInterface;
+
+/**
+ The unprivileged interface this object exports. (server)
  */
-@property(retain, nullable) NSXPCInterface *exportedInterface;
+@property(retain, nullable) NSXPCInterface *unprivilegedExportedInterface;
```

Comment:
This interface is nice and clean but to avoid breaking upgrading clients, I'd like to leave the exportedInterface property around for a release or two. Could you add it back and mark it as deprecated, like so:

```objc
/**
  Old interface property, please update to use privilegedExportedInterface and/or unprivilegedExportedInterface instead.
*/
@property(retain, nullable) NSXPCInterface *exportedInterface __attribute__((
   deprecated(Use privilegedExportedInterface and / or unprivilegedExportedInterface instead.)));
```